book/analytics/client_side_api/client_side_api.md

```diff
@@ -59,49 +62,53 @@ Some characteristics of users should be stored as user properties[c]. This is pa
 Properties let us easily view the distribution of Event characteristics so we can answer questions such as "who is the most read author?" or "what percentage of users reading articles are registered?"
 
 ### Android:
-
-// Capture author info & user status  
-Map<String, String> articleParams = new HashMap<String, String>();  
-//param keys and values have to be of String type  
-articleParams.put("Author", "John Q");  
-articleParams.put("User Status", "Registered");  
-// sending event parameter  
-AnalyticsAgentUtil.logEvent("Article Read", articleParams);
+```
+  // Capture author info & user status  
+  Map<String, String> articleParams = new HashMap<String, String>();  
+  //param keys and values have to be of String type  
+  articleParams.put("Author", "John Q");  
+  articleParams.put("User Status", "Registered");  
+  // sending event parameter  
+  AnalyticsAgentUtil.logEvent("Article Read", articleParams);
+```
 
 ### iOS:
-
-NSDictionary \*analyticsDictionary = @{@”Author” : @”"John Q", @User Status": @"Registered"};  
-[APAnalyticsManager trackEvent:@"Article Read" withParameters:analyticsDictionary];
+```
+  NSDictionary \*analyticsDictionary = @{@”Author” : @”"John Q", @User Status": @"Registered"};  
```

Comment:
is the `\` in the `NSDictionary \*analyticsDictionary` is necessary?

Reply:
👍